thousandeyes/schemas.go

```diff
@@ -397,6 +425,12 @@ var schemas = map[string]*schema.Schema{
 		Description: "Type of test",
 		Computed:  true,
 	},
+	"type--label": {
```

Comment:
same as above?

Reply:
yea, I agree the double hyphen seems odd